grpc-runtime/src/main/scala/monix/grpc/runtime/server/ServerCallHandlers.scala

```diff
@@ -155,6 +244,38 @@ object ServerCallHandlers {
     }
   }
 
+  /**
+   * Defines a grpc service call handler that receives several requests request
+   * from the client and returns several responses from the server.
+   *
+   * @param f is the function that turns a request and metadata into a response.
+   * @param options is the configuration to configure options for this call.
+   * @param scheduler is the (implicit) scheduler available in the service definition.
+   * @return a grpc server call handler that will be responsible for processing this call.
+   */
+  def streamingToStreamingCallWithoutMetaData[T, R](
+      f: (Observable[T]) => Observable[R],
+      options: ServerCallOptions = ServerCallOptions.default
```

Comment:
adds special constructors to these classes to have lambdas without MetaData, it isn't possible to have methods with the same name because the function type is generic.